### PR TITLE
Fix makefile and scripts

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- Fix for prow

**Related issue(s)**
Resolves kyma-project/test-infra#156

### CODE REVIEW

File: scripts/helpers/git-config.sh [hunk 1]
```diff
@@ -18,7 +18,6 @@ done
 readonly SSH_FILE
 
 # configure git
-git config --global user.email "kyma.bot@sap.com" || exit
-git config --global user.name "Kyma Bot" || exit
-
-git config --global core.sshCommand 'ssh -i '$SSH_FILE'' || exit
+git config --global user.email "kyma.bot@sap.com"
```
Comment (michal-hudy) [inline scripts/helpers/git-config.sh]: Define email and name of `kyma-bot` as preset in Prow.

File: scripts/generate-docs.sh [hunk 2]
```diff
@@ -83,10 +83,28 @@ generate() {
 
 publish() {
     if [[ -n ${SSH_FILE} ]]; then
-        echo "Configuring git"
-        bash "${SCRIPTS_DIR}/helpers/git-config.sh" --ssh-file "${SSH_FILE}" || return
-    fi
+        # make ssh dir
+        mkdir /root/.ssh/
```
Comment (michal-hudy) [inline scripts/generate-docs.sh]: Such logic should be rather in `git-config.sh` as it is also used in `prepare-website.sh`.

File: scripts/generate-docs.sh [hunk 3]
```diff
@@ -96,12 +114,13 @@ publish() {
         return
     fi
 
+
     echo "Commit documentation"
-    git add "${DOCUMENTATION_DIR}" || return
```
Comment (michal-hudy) [inline scripts/generate-docs.sh]: Why did you remove it? It should only add changes from that directory.

File: scripts/generate-docs.sh [hunk 4]
```diff
@@ -96,12 +114,13 @@ publish() {
...
-    git commit -m "Publish documentation for Kyma" --no-verify || return
 
+    git remote add origin git@github.com:kyma-project/website.git
```
Comment (michal-hudy) [inline scripts/generate-docs.sh]: It should be also defined in `git-config.sh`.

File: scripts/generate-docs.sh (hunk 2 above)
Comment (akucharska) [reply scripts/generate-docs.sh]: not all of it

File: scripts/generate-docs.sh [hunk 5]
```diff
@@ -83,10 +83,28 @@ generate() {
...
 publish() {
     if [[ -n ${SSH_FILE} ]]; then
-        echo "Configuring git"
-        bash "${SCRIPTS_DIR}/helpers/git-config.sh" --ssh-file "${SSH_FILE}" || return
-    fi
+        # make ssh dir
+        mkdir /root/.ssh/
+
+        # create known_hosts file
+        touch /root/.ssh/known_hosts
```
Comment (michal-hudy) [inline scripts/generate-docs.sh]: It shouldn't be fixed to `root`, it should work for the current user.
